pubmed_parser/medline_parser.py

```diff
@@ -472,7 +472,7 @@ def parse_references(pubmed_article, reference_list):
         return references
     else:
         references = ";".join(
-            [ref["pmid"] for ref in references if ref["pmid"] is not ""]
+            [ref["pmid"] for ref in references if ref["pmid"] != ""]
```

Comment:
```python
 [ref["pmid"] for ref in references if ref["pmid"]]
```
Empty strings are "falsy".  In an `if` statement, you can drop the explicit test for not empty string.
https://stackoverflow.com/questions/9573244/how-to-check-if-the-string-is-empty

Reply:
Thanks for the reminder. I think it also depends on your style: more pythonic or more explicit. Feel free to change this.

Reply:
Thanks for the change!